lua/lang/dart.lua

```diff
@@ -37,6 +40,10 @@ M.lsp = function()
     return
   end
 
+  if O.lang.dart.flutter_tools.active then
```

Comment:
how does the flutter lang server start?

Reply:
for instance we start separate ones for rust-tools and java-tools

Reply:
rust-tools draws inspiration from akinsho/flutter-tools.nvim
"You can override any options available in the lspconfig setup, this call essentially wraps it and adds some extra flutter specific handlers and utilisation options."
https://github.com/akinsho/flutter-tools.nvim/blob/main/lua/flutter-tools/lsp.lua